tests/dummy/app/controllers/index.js

```diff
@@ -72,14 +104,40 @@ export default Controller.extend({
   networkFailure : false,
 
   actions : {
-    dragEndAction ({sourceList, sourceIndex, targetList, targetIndex}) {
+    dragEnd ({sourceList, sourceIndex, targetList, targetIndex}) {
       if (sourceList === targetList && sourceIndex === targetIndex) return
 
       const item = sourceList.objectAt(sourceIndex)
 
       sourceList.removeAt(sourceIndex)
       targetList.insertAt(targetIndex, item)
     },
+
+    determineForeignPosition ({draggedItem, items}) {
+      items = A(items.slice()) // create a copy of the list
```

Comment:
How about writing this as:
```
return items
  .toArray() // create a copy of the list, convert to Ember.Array if needed: `Ember.Array(items.slice())`
  .addObject(draggedItem)
  .sortBy('name')
  .indexOf(draggedItem)
```
The snippet in the `README.md` could be updated as well.